coverage_ai/AICaller.py

```
@@ -5,7 +5,14 @@
 import litellm
 from functools import wraps
 from wandb.sdk.data_types.trace_tree import Trace
-from tenacity import retry, retry_if_exception_type, retry_if_not_exception_type, stop_after_attempt, wait_fixed
+from tenacity import (
+    retry,
+    retry_if_exception_type,
```

Ruff (0.8.0)

`tenacity.retry_if_exception_type` imported but unused

Remove unused import

(F401)